dockerfiles/theia-dev/e2e/Dockerfile

```diff
@@ -13,7 +13,7 @@ ARG GITHUB_TOKEN=''
 ENV GITHUB_TOKEN=$GITHUB_TOKEN
 
 # Just try to build the latest theia with current image
-RUN git clone -b 'master' --single-branch --depth 1 https://github.com/eclipse-theia/theia theia
+RUN git clone -b ${GIT_BRANCH_NAME} --single-branch --depth 1 https://github.com/eclipse-theia/theia theia
```

Comment:
it's odd... build is failed on CI but Docker build is passed on my side with the correct value substituted
I'll figure it out tomorrow